pygmt/helpers/utils.py

```diff
@@ -15,127 +15,154 @@
 from pygmt.exceptions import GMTInvalidInput
 
 
-def _validate_data_input(
-    data=None, x=None, y=None, z=None, required_z=False, required_data=True, kind=None
+def validate_data_input(
```

Comment:
Any reason to remove the leading underscore from `_validate_data_input`? Do we want to make this function more public?

Reply:
Because this function is now called in `pygmt/clib/sessions.py`.